{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/cli.py

```diff
@@ -0,0 +1,85 @@
+import pathlib
+import time
+import logging
+
+import click
+from fastapi import FastAPI, Request
```

Comment:
logic should be separated from the command-line, which should only be about processing argument and calling the matching function